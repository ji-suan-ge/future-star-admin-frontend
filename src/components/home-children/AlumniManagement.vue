<template>
  <el-container>
    <el-main>
      <el-row>
        <el-col :span="4" :offset="3">
          <el-select class="selection" v-model="semester" @change="findAllClass()" filterable placeholder="请选择学期">
            <el-option
              v-for="semesters in semesterOptions"
              :key="semesters.id"
              :label="semesters.label"
              :value="semesters.semester">
            </el-option>
          </el-select>
        </el-col>
        <el-col :span="4">
          <el-select class="selection" v-model="clazz" :disabled="class_state" @change="search()" filterable placeholder="请选择班级">
            <el-option
              v-for="classes in clazzOptions"
              :key="classes.clazz"
              :label="classes.label"
              :value="classes.clazz">
            </el-option>
          </el-select>
        </el-col>
        <el-col :span="6">
          <el-input class='search-input' v-model="keyword" @change="search()" placeholder="请输入姓名"/>
        </el-col>
        <el-col :span="4" :offset="1">
          <el-button type="primary" icon="el-icon-search" @click="search()" round>搜索</el-button>
        </el-col>
      </el-row>
      <el-divider/>
      <el-table
        :data="tableData"
        v-loading="loading"
        stripe
        class="info-table">
        <el-table-column
          prop="name"
          align="center"
          label="姓名">
        </el-table-column>
        <el-table-column
          prop="gender"
          align="center"
          label="性别">
        </el-table-column>
        <el-table-column
          prop="birthday"
          align="center"
          label="出生日期">
        </el-table-column>
        <el-table-column
          prop="phone_number"
          align="center"
          label="联系电话">
        </el-table-column>
        <el-table-column
          prop="wx"
          align="center"
          label="微信号">
        </el-table-column>
        <el-table-column
          prop="email"
          align="center"
          label="邮箱">
        </el-table-column>
        <el-table-column
          prop="city"
          align="center"
          label="所在城市">
        </el-table-column>
        <el-table-column
          align="center"
          label="全部信息">
          <template slot-scope="scope">
            <el-button
              type="primary"
              size="small"
              @click="showAllInfo(scope.row)"
              round>
              详情
            </el-button>
          </template>
        </el-table-column>
        <el-table-column
          align="center"
          label="操作">
          <template slot-scope="scope">
            <el-button
              type="danger"
              size="small"
              @click="deleteAlumni(scope.row, scope.$index)"
              round>
              删除
            </el-button>
          </template>
        </el-table-column>
      </el-table>
    </el-main>
    <el-footer>
      <el-pagination
        class="pagination"
        @current-change="handleCurrentChange"
        :current-page="currentAlumniPage"
        layout="prev, pager, next"
        :page-count="totalPage">
      </el-pagination>
    </el-footer>
  </el-container>
</template>

<script>
export default {
  name: 'AlumniManagementPage',
  data () {
    return {
      currentAlumniPage: 1,
      currentSemesterPage: 1,
      currentClassPage: 1,
      totalPage: 1,
      page_size: 20,
      class_state: true,
      loading: true,
      keyword: null,
      clazz: null,
      semester: null,
      tableData: [],
      semesterOptions: [],
      clazzOptions: []
    }
  },
  created () {
    let that = this
    if (that.axios) {
      that.findAllAlumni()
      that.findAllSemester()
    }
  },
  methods: {
    findAllAlumni () {
      let that = this
      let params = {
        page_size: that.page_size,
        page: that.currentAlumniPage
      }
      that.axios.get('/student/student', { params }).then((response) => {
        if (response.data.code === '2000') {
          that.tableData = response.data.data.results
          for (let i = 0; i < that.tableData.length; i++) {
            if (that.tableData[i].gender === 0) {
              that.tableData[i].gender = '男'
            } else {
              that.tableData[i].gender = '女'
            }
          }
          that.loading = false
          that.$forceUpdate()
        } else {
          that.$message({
            type: 'error',
            message: '服务器繁忙，请稍候重试',
            duration: 2000
          })
        }
      }).catch(function (error) {
        console.log(error)
        that.$message({
          type: 'error',
          message: '服务器繁忙，请稍候重试',
          duration: 2000
        })
      })
    },
    findAllSemester () {
      let that = this
      let params = {
        page_size: that.page_size,
        page: that.currentSemesterPage
      }
      that.axios.get('/semester/semester', { params }).then(function (response) {
        if (response.data.code === '2000') {
          let results = response.data.data.results
          that.semesterOptions[0] = {
            semester: null,
            label: '全部学期',
            id: 0
          }
          for (let i = 1; i < results.length + 1; i++) {
            that.semesterOptions[i] = {
              semester: results[i - 1]['period_semester'],
              label: results[i - 1]['subject'],
              id: results[i - 1]['id']
            }
          }
          that.$forceUpdate()
        } else {
          that.$message({
            type: 'error',
            message: '服务器繁忙，请稍候重试',
            duration: 2000
          })
        }
      }).catch(function (error) {
        console.log(error)
        that.$message({
          type: 'error',
          message: '服务器繁忙，请稍候重试',
          duration: 2000
        })
      })
    },
    findAllClass () {
      let that = this
      that.clazzOptions = []
      that.clazz = null
      if (that.semester === null) {
        that.class_state = true
      } else {
        that.class_state = false
      }
      let params = {
        page_size: that.page_size,
        page: that.currentClassPage,
        semester_id: that.semester
      }
      that.axios.get('/clazz/clazz', { params }).then(function (response) {
        if (response.data.code === '2000') {
          let results = response.data.data.results
          for (let i = 0; i < results.length; i++) {
            that.clazzOptions[i] = {
              clazz: results[i]['id'],
              label: results[i]['name']
            }
          }
          that.search()
          that.$forceUpdate()
        } else {
          that.$message({
            type: 'error',
            message: '服务器繁忙，请稍候重试',
            duration: 2000
          })
        }
      }).catch(function (error) {
        console.log(error)
        that.$message({
          type: 'error',
          message: '服务器繁忙，请稍候重试',
          duration: 2000
        })
      })
    },
    search () {
      let that = this
      let params = {
        page_size: that.page_size,
        page: that.currentAlumniPage,
        name: that.keyword,
        semester_id: that.semester,
        clazz_id: that.clazz
      }
      that.axios.get('/student/student', { params }).then((response) => {
        that.tableData = response.data.data.results
        for (let i = 0; i < that.tableData.length; i++) {
          if (that.tableData[i].gender === 0) {
            that.tableData[i].gender = '男'
          } else {
            that.tableData[i].gender = '女'
          }
        }
        that.loading = false
        that.$forceUpdate()
      })
    },
    handleCurrentChange (currentAlumniPage) {
      this.currentAlumniPage = currentAlumniPage
      this.findAllAlumni()
    },
    deleteAlumni (student, index) {
      let that = this
      let url = '/student/student/' + student.id
      that.axios.delete(url
      ).then((response) => {
        if (response.data.code === '2000') {
          that.tableData.splice(index, 1)
          that.$message({
            type: 'success',
            message: '删除成功',
            duration: 2000
          })
        } else {
          that.$message({
            type: 'error',
            message: '服务器繁忙，请稍候重试',
            duration: 2000
          })
        }
      }).catch((error) => {
        console.log(error)
        that.$message({
          type: 'error',
          message: '服务器繁忙，请稍候重试',
          duration: 2000
        })
      })
    },
    showAllInfo (alumniInfo) {
      this.$store.dispatch('changeInfoOfAnAlumni', alumniInfo)
      this.$router.push('/show-all-information')
    }
  }
}
</script>

<style scoped>
  .info-table {
    margin: auto;
  }

  .selection {
    margin-right: 20px;
  }

  .search-input {
    display: inline-block;
  }

  .pagination {
    text-align: center;
  }
</style>
