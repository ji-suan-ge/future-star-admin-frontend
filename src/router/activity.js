import CreateActivityPage from '../page/CreateActivityPage'
import ShowActivityParticipantsPage from '../page/ShowActivityParticipantsPage'
import EditExistingActivityPage from '../page/EditExistingActivityPage'
import ShowActivityInformationPage from '../page/ShowActivityInformationPage'

export default [
  {
    path: '/create-activity',
    name: 'CreateActivityPage',
    component: CreateActivityPage
  },
  {
    path: '/show-participants',
    name: 'ShowActivityParticipantsPage',
    component: ShowActivityParticipantsPage
  },
  {
    path: '/edit-activity',
    name: 'EditExistingActivityPage',
    component: EditExistingActivityPage
  },
  {
    path: '/show-activity-details',
    name: 'ShowActivityInformationPage',
    component: ShowActivityInformationPage
  }
]
