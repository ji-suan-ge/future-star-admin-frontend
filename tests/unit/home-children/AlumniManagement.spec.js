import { createLocalVue, mount } from '@vue/test-utils'
import AlumniManagement from '../../../src/components/home-children/AlumniManagement'
import Vuex from 'vuex'
import ElementUI from 'element-ui'
import VueRouter from 'vue-router'
import store from '../../../src/store'
import router from '../../../src/router'

describe('AlumniManagement', () => {
  const localVue = createLocalVue()
  localVue.use(Vuex)
  localVue.use(ElementUI)
  localVue.use(VueRouter)
  const wrapper = mount(AlumniManagement, {
    store,
    router,
    localVue
  })
  test('快照测试', async () => {
    await expect(wrapper.html()).toMatchSnapshot()
  })
  test('是一个 Vue 实例', async () => {
    await expect(wrapper.isVueInstance()).toBeTruthy()
  })
})
