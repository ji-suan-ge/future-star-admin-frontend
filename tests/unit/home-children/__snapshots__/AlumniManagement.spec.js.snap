// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AlumniManagement 快照测试 1`] = `
<section class="el-container is-vertical">
  <main class="el-main">
    <div class="el-row">
      <div class="el-col el-col-4">
        <div class="el-select selection">
          <!---->
          <div class="el-input el-input--suffix">
            <!----><input type="text" readonly="readonly" autocomplete="off" placeholder="请选择学期" class="el-input__inner">
            <!----><span class="el-input__suffix"><span class="el-input__suffix-inner"><i class="el-select__caret el-input__icon el-icon-arrow-up"></i><!----><!----><!----><!----><!----></span>
            <!----></span>
            <!---->
            <!---->
          </div>
          <div class="el-select-dropdown el-popper" style="display: none;">
            <div class="el-scrollbar" style="">
              <div class="el-select-dropdown__wrap el-scrollbar__wrap el-scrollbar__wrap--hidden-default">
                <ul class="el-scrollbar__view el-select-dropdown__list">
                  <!---->
                  <li class="el-select-dropdown__item"><span>第一期</span></li>
                  <li class="el-select-dropdown__item"><span>第二期</span></li>
                  <li class="el-select-dropdown__item"><span>第三期</span></li>
                </ul>
              </div>
              <div class="el-scrollbar__bar is-horizontal">
                <div class="el-scrollbar__thumb" style="transform: translateX(0%); webkit-transform: translateX(0%);"></div>
              </div>
              <div class="el-scrollbar__bar is-vertical">
                <div class="el-scrollbar__thumb" style="transform: translateY(0%); webkit-transform: translateY(0%);"></div>
              </div>
            </div>
            <!---->
          </div>
        </div>
      </div>
      <div class="el-col el-col-4">
        <div class="el-select selection">
          <!---->
          <div class="el-input el-input--suffix">
            <!----><input type="text" readonly="readonly" autocomplete="off" placeholder="请选择班级" class="el-input__inner">
            <!----><span class="el-input__suffix"><span class="el-input__suffix-inner"><i class="el-select__caret el-input__icon el-icon-arrow-up"></i><!----><!----><!----><!----><!----></span>
            <!----></span>
            <!---->
            <!---->
          </div>
          <div class="el-select-dropdown el-popper" style="display: none;">
            <div class="el-scrollbar" style="">
              <div class="el-select-dropdown__wrap el-scrollbar__wrap el-scrollbar__wrap--hidden-default">
                <ul class="el-scrollbar__view el-select-dropdown__list">
                  <!---->
                  <li class="el-select-dropdown__item"><span>素质教育专题班</span></li>
                  <li class="el-select-dropdown__item"><span>教育信息化专题班</span></li>
                </ul>
              </div>
              <div class="el-scrollbar__bar is-horizontal">
                <div class="el-scrollbar__thumb" style="transform: translateX(0%); webkit-transform: translateX(0%);"></div>
              </div>
              <div class="el-scrollbar__bar is-vertical">
                <div class="el-scrollbar__thumb" style="transform: translateY(0%); webkit-transform: translateY(0%);"></div>
              </div>
            </div>
            <!---->
          </div>
        </div>
      </div>
      <div class="el-col el-col-8">
        <div class="search-input el-input">
          <!----><input type="text" autocomplete="off" placeholder="请输入搜索内容" class="el-input__inner">
          <!---->
          <!---->
          <!---->
          <!---->
        </div>
      </div>
      <div class="el-col el-col-4"><button type="button" class="el-button el-button--primary">
          <!----><i class="el-icon-search"></i><span>搜索</span></button></div>
    </div>
    <div class="el-divider el-divider--horizontal">
      <!---->
    </div>
    <div class="el-table el-table--fit el-table--striped el-table--scrollable-x el-table--enable-row-hover el-table--enable-row-transition">
      <div class="hidden-columns">
        <div></div>
        <div></div>
        <div></div>
        <div></div>
        <div></div>
        <div></div>
        <div></div>
        <div></div>
      </div>
      <div class="el-table__header-wrapper">
        <table cellspacing="0" cellpadding="0" border="0" class="el-table__header" style="width: 1180px;">
          <colgroup>
            <col name="el-table_1_column_1" width="115">
            <col name="el-table_1_column_2" width="115">
            <col name="el-table_1_column_3" width="170">
            <col name="el-table_1_column_4" width="170">
            <col name="el-table_1_column_5" width="170">
            <col name="el-table_1_column_6" width="210">
            <col name="el-table_1_column_7" width="115">
            <col name="el-table_1_column_8" width="115">
          </colgroup>
          <thead class="">
            <tr class="">
              <th colspan="1" rowspan="1" class="el-table_1_column_1  is-center   is-leaf">
                <div class="cell">姓名</div>
              </th>
              <th colspan="1" rowspan="1" class="el-table_1_column_2  is-center   is-leaf">
                <div class="cell">性别</div>
              </th>
              <th colspan="1" rowspan="1" class="el-table_1_column_3  is-center   is-leaf">
                <div class="cell">出生日期</div>
              </th>
              <th colspan="1" rowspan="1" class="el-table_1_column_4  is-center   is-leaf">
                <div class="cell">联系电话</div>
              </th>
              <th colspan="1" rowspan="1" class="el-table_1_column_5  is-center   is-leaf">
                <div class="cell">微信号</div>
              </th>
              <th colspan="1" rowspan="1" class="el-table_1_column_6  is-center   is-leaf">
                <div class="cell">邮箱</div>
              </th>
              <th colspan="1" rowspan="1" class="el-table_1_column_7  is-center   is-leaf">
                <div class="cell">所在城市</div>
              </th>
              <th colspan="1" rowspan="1" class="el-table_1_column_8  is-center   is-leaf">
                <div class="cell">操作</div>
              </th>
            </tr>
          </thead>
        </table>
      </div>
      <div class="el-table__body-wrapper is-scrolling-left">
        <table cellspacing="0" cellpadding="0" border="0" class="el-table__body" style="width: 1180px;">
          <colgroup>
            <col name="el-table_1_column_1" width="115">
            <col name="el-table_1_column_2" width="115">
            <col name="el-table_1_column_3" width="170">
            <col name="el-table_1_column_4" width="170">
            <col name="el-table_1_column_5" width="170">
            <col name="el-table_1_column_6" width="210">
            <col name="el-table_1_column_7" width="115">
            <col name="el-table_1_column_8" width="115">
          </colgroup>
          <tbody>
            <tr class="el-table__row">
              <td rowspan="1" colspan="1" class="el-table_1_column_1 is-center ">
                <div class="cell">张邦鑫</div>
              </td>
              <td rowspan="1" colspan="1" class="el-table_1_column_2 is-center ">
                <div class="cell">男</div>
              </td>
              <td rowspan="1" colspan="1" class="el-table_1_column_3 is-center ">
                <div class="cell">1980/10/01</div>
              </td>
              <td rowspan="1" colspan="1" class="el-table_1_column_4 is-center ">
                <div class="cell">12345678901</div>
              </td>
              <td rowspan="1" colspan="1" class="el-table_1_column_5 is-center ">
                <div class="cell">1</div>
              </td>
              <td rowspan="1" colspan="1" class="el-table_1_column_6 is-center ">
                <div class="cell">edstars@100tal.com</div>
              </td>
              <td rowspan="1" colspan="1" class="el-table_1_column_7 is-center ">
                <div class="cell">北京</div>
              </td>
              <td rowspan="1" colspan="1" class="el-table_1_column_8 is-center ">
                <div class="cell"><button type="button" class="el-button el-button--primary el-button--small is-circle">
                    <!----><i class="el-icon-more"></i>
                    <!----></button> <button type="button" class="el-button el-button--danger el-button--small is-circle">
                    <!----><i class="el-icon-delete"></i>
                    <!----></button></div>
              </td>
            </tr>
            <tr class="el-table__row el-table__row--striped">
              <td rowspan="1" colspan="1" class="el-table_1_column_1 is-center ">
                <div class="cell">张邦鑫</div>
              </td>
              <td rowspan="1" colspan="1" class="el-table_1_column_2 is-center ">
                <div class="cell">男</div>
              </td>
              <td rowspan="1" colspan="1" class="el-table_1_column_3 is-center ">
                <div class="cell">1980/10/01</div>
              </td>
              <td rowspan="1" colspan="1" class="el-table_1_column_4 is-center ">
                <div class="cell">12345678901</div>
              </td>
              <td rowspan="1" colspan="1" class="el-table_1_column_5 is-center ">
                <div class="cell">2</div>
              </td>
              <td rowspan="1" colspan="1" class="el-table_1_column_6 is-center ">
                <div class="cell">edstars@100tal.com</div>
              </td>
              <td rowspan="1" colspan="1" class="el-table_1_column_7 is-center ">
                <div class="cell">北京</div>
              </td>
              <td rowspan="1" colspan="1" class="el-table_1_column_8 is-center ">
                <div class="cell"><button type="button" class="el-button el-button--primary el-button--small is-circle">
                    <!----><i class="el-icon-more"></i>
                    <!----></button> <button type="button" class="el-button el-button--danger el-button--small is-circle">
                    <!----><i class="el-icon-delete"></i>
                    <!----></button></div>
              </td>
            </tr>
            <tr class="el-table__row">
              <td rowspan="1" colspan="1" class="el-table_1_column_1 is-center ">
                <div class="cell">张邦鑫</div>
              </td>
              <td rowspan="1" colspan="1" class="el-table_1_column_2 is-center ">
                <div class="cell">男</div>
              </td>
              <td rowspan="1" colspan="1" class="el-table_1_column_3 is-center ">
                <div class="cell">1980/10/01</div>
              </td>
              <td rowspan="1" colspan="1" class="el-table_1_column_4 is-center ">
                <div class="cell">12345678901</div>
              </td>
              <td rowspan="1" colspan="1" class="el-table_1_column_5 is-center ">
                <div class="cell">3</div>
              </td>
              <td rowspan="1" colspan="1" class="el-table_1_column_6 is-center ">
                <div class="cell">edstars@100tal.com</div>
              </td>
              <td rowspan="1" colspan="1" class="el-table_1_column_7 is-center ">
                <div class="cell">北京</div>
              </td>
              <td rowspan="1" colspan="1" class="el-table_1_column_8 is-center ">
                <div class="cell"><button type="button" class="el-button el-button--primary el-button--small is-circle">
                    <!----><i class="el-icon-more"></i>
                    <!----></button> <button type="button" class="el-button el-button--danger el-button--small is-circle">
                    <!----><i class="el-icon-delete"></i>
                    <!----></button></div>
              </td>
            </tr>
            <tr class="el-table__row el-table__row--striped">
              <td rowspan="1" colspan="1" class="el-table_1_column_1 is-center ">
                <div class="cell">张邦鑫</div>
              </td>
              <td rowspan="1" colspan="1" class="el-table_1_column_2 is-center ">
                <div class="cell">男</div>
              </td>
              <td rowspan="1" colspan="1" class="el-table_1_column_3 is-center ">
                <div class="cell">1980/10/01</div>
              </td>
              <td rowspan="1" colspan="1" class="el-table_1_column_4 is-center ">
                <div class="cell">12345678901</div>
              </td>
              <td rowspan="1" colspan="1" class="el-table_1_column_5 is-center ">
                <div class="cell">4</div>
              </td>
              <td rowspan="1" colspan="1" class="el-table_1_column_6 is-center ">
                <div class="cell">edstars@100tal.com</div>
              </td>
              <td rowspan="1" colspan="1" class="el-table_1_column_7 is-center ">
                <div class="cell">北京</div>
              </td>
              <td rowspan="1" colspan="1" class="el-table_1_column_8 is-center ">
                <div class="cell"><button type="button" class="el-button el-button--primary el-button--small is-circle">
                    <!----><i class="el-icon-more"></i>
                    <!----></button> <button type="button" class="el-button el-button--danger el-button--small is-circle">
                    <!----><i class="el-icon-delete"></i>
                    <!----></button></div>
              </td>
            </tr>
            <tr class="el-table__row">
              <td rowspan="1" colspan="1" class="el-table_1_column_1 is-center ">
                <div class="cell">张邦鑫</div>
              </td>
              <td rowspan="1" colspan="1" class="el-table_1_column_2 is-center ">
                <div class="cell">男</div>
              </td>
              <td rowspan="1" colspan="1" class="el-table_1_column_3 is-center ">
                <div class="cell">1980/10/01</div>
              </td>
              <td rowspan="1" colspan="1" class="el-table_1_column_4 is-center ">
                <div class="cell">12345678901</div>
              </td>
              <td rowspan="1" colspan="1" class="el-table_1_column_5 is-center ">
                <div class="cell">5</div>
              </td>
              <td rowspan="1" colspan="1" class="el-table_1_column_6 is-center ">
                <div class="cell">edstars@100tal.com</div>
              </td>
              <td rowspan="1" colspan="1" class="el-table_1_column_7 is-center ">
                <div class="cell">北京</div>
              </td>
              <td rowspan="1" colspan="1" class="el-table_1_column_8 is-center ">
                <div class="cell"><button type="button" class="el-button el-button--primary el-button--small is-circle">
                    <!----><i class="el-icon-more"></i>
                    <!----></button> <button type="button" class="el-button el-button--danger el-button--small is-circle">
                    <!----><i class="el-icon-delete"></i>
                    <!----></button></div>
              </td>
            </tr>
            <tr class="el-table__row el-table__row--striped">
              <td rowspan="1" colspan="1" class="el-table_1_column_1 is-center ">
                <div class="cell">张邦鑫</div>
              </td>
              <td rowspan="1" colspan="1" class="el-table_1_column_2 is-center ">
                <div class="cell">男</div>
              </td>
              <td rowspan="1" colspan="1" class="el-table_1_column_3 is-center ">
                <div class="cell">1980/10/01</div>
              </td>
              <td rowspan="1" colspan="1" class="el-table_1_column_4 is-center ">
                <div class="cell">12345678901</div>
              </td>
              <td rowspan="1" colspan="1" class="el-table_1_column_5 is-center ">
                <div class="cell">6</div>
              </td>
              <td rowspan="1" colspan="1" class="el-table_1_column_6 is-center ">
                <div class="cell">edstars@100tal.com</div>
              </td>
              <td rowspan="1" colspan="1" class="el-table_1_column_7 is-center ">
                <div class="cell">北京</div>
              </td>
              <td rowspan="1" colspan="1" class="el-table_1_column_8 is-center ">
                <div class="cell"><button type="button" class="el-button el-button--primary el-button--small is-circle">
                    <!----><i class="el-icon-more"></i>
                    <!----></button> <button type="button" class="el-button el-button--danger el-button--small is-circle">
                    <!----><i class="el-icon-delete"></i>
                    <!----></button></div>
              </td>
            </tr>
            <tr class="el-table__row">
              <td rowspan="1" colspan="1" class="el-table_1_column_1 is-center ">
                <div class="cell">张邦鑫</div>
              </td>
              <td rowspan="1" colspan="1" class="el-table_1_column_2 is-center ">
                <div class="cell">男</div>
              </td>
              <td rowspan="1" colspan="1" class="el-table_1_column_3 is-center ">
                <div class="cell">1980/10/01</div>
              </td>
              <td rowspan="1" colspan="1" class="el-table_1_column_4 is-center ">
                <div class="cell">12345678901</div>
              </td>
              <td rowspan="1" colspan="1" class="el-table_1_column_5 is-center ">
                <div class="cell">7</div>
              </td>
              <td rowspan="1" colspan="1" class="el-table_1_column_6 is-center ">
                <div class="cell">edstars@100tal.com</div>
              </td>
              <td rowspan="1" colspan="1" class="el-table_1_column_7 is-center ">
                <div class="cell">北京</div>
              </td>
              <td rowspan="1" colspan="1" class="el-table_1_column_8 is-center ">
                <div class="cell"><button type="button" class="el-button el-button--primary el-button--small is-circle">
                    <!----><i class="el-icon-more"></i>
                    <!----></button> <button type="button" class="el-button el-button--danger el-button--small is-circle">
                    <!----><i class="el-icon-delete"></i>
                    <!----></button></div>
              </td>
            </tr>
            <tr class="el-table__row el-table__row--striped">
              <td rowspan="1" colspan="1" class="el-table_1_column_1 is-center ">
                <div class="cell">张邦鑫</div>
              </td>
              <td rowspan="1" colspan="1" class="el-table_1_column_2 is-center ">
                <div class="cell">男</div>
              </td>
              <td rowspan="1" colspan="1" class="el-table_1_column_3 is-center ">
                <div class="cell">1980/10/01</div>
              </td>
              <td rowspan="1" colspan="1" class="el-table_1_column_4 is-center ">
                <div class="cell">12345678901</div>
              </td>
              <td rowspan="1" colspan="1" class="el-table_1_column_5 is-center ">
                <div class="cell">8</div>
              </td>
              <td rowspan="1" colspan="1" class="el-table_1_column_6 is-center ">
                <div class="cell">edstars@100tal.com</div>
              </td>
              <td rowspan="1" colspan="1" class="el-table_1_column_7 is-center ">
                <div class="cell">北京</div>
              </td>
              <td rowspan="1" colspan="1" class="el-table_1_column_8 is-center ">
                <div class="cell"><button type="button" class="el-button el-button--primary el-button--small is-circle">
                    <!----><i class="el-icon-more"></i>
                    <!----></button> <button type="button" class="el-button el-button--danger el-button--small is-circle">
                    <!----><i class="el-icon-delete"></i>
                    <!----></button></div>
              </td>
            </tr>
            <tr class="el-table__row">
              <td rowspan="1" colspan="1" class="el-table_1_column_1 is-center ">
                <div class="cell">张邦鑫</div>
              </td>
              <td rowspan="1" colspan="1" class="el-table_1_column_2 is-center ">
                <div class="cell">男</div>
              </td>
              <td rowspan="1" colspan="1" class="el-table_1_column_3 is-center ">
                <div class="cell">1980/10/01</div>
              </td>
              <td rowspan="1" colspan="1" class="el-table_1_column_4 is-center ">
                <div class="cell">12345678901</div>
              </td>
              <td rowspan="1" colspan="1" class="el-table_1_column_5 is-center ">
                <div class="cell">9</div>
              </td>
              <td rowspan="1" colspan="1" class="el-table_1_column_6 is-center ">
                <div class="cell">edstars@100tal.com</div>
              </td>
              <td rowspan="1" colspan="1" class="el-table_1_column_7 is-center ">
                <div class="cell">北京</div>
              </td>
              <td rowspan="1" colspan="1" class="el-table_1_column_8 is-center ">
                <div class="cell"><button type="button" class="el-button el-button--primary el-button--small is-circle">
                    <!----><i class="el-icon-more"></i>
                    <!----></button> <button type="button" class="el-button el-button--danger el-button--small is-circle">
                    <!----><i class="el-icon-delete"></i>
                    <!----></button></div>
              </td>
            </tr>
            <tr class="el-table__row el-table__row--striped">
              <td rowspan="1" colspan="1" class="el-table_1_column_1 is-center ">
                <div class="cell">张邦鑫</div>
              </td>
              <td rowspan="1" colspan="1" class="el-table_1_column_2 is-center ">
                <div class="cell">男</div>
              </td>
              <td rowspan="1" colspan="1" class="el-table_1_column_3 is-center ">
                <div class="cell">1980/10/01</div>
              </td>
              <td rowspan="1" colspan="1" class="el-table_1_column_4 is-center ">
                <div class="cell">12345678901</div>
              </td>
              <td rowspan="1" colspan="1" class="el-table_1_column_5 is-center ">
                <div class="cell">10</div>
              </td>
              <td rowspan="1" colspan="1" class="el-table_1_column_6 is-center ">
                <div class="cell">edstars@100tal.com</div>
              </td>
              <td rowspan="1" colspan="1" class="el-table_1_column_7 is-center ">
                <div class="cell">北京</div>
              </td>
              <td rowspan="1" colspan="1" class="el-table_1_column_8 is-center ">
                <div class="cell"><button type="button" class="el-button el-button--primary el-button--small is-circle">
                    <!----><i class="el-icon-more"></i>
                    <!----></button> <button type="button" class="el-button el-button--danger el-button--small is-circle">
                    <!----><i class="el-icon-delete"></i>
                    <!----></button></div>
              </td>
            </tr>
            <tr class="el-table__row">
              <td rowspan="1" colspan="1" class="el-table_1_column_1 is-center ">
                <div class="cell">张邦鑫</div>
              </td>
              <td rowspan="1" colspan="1" class="el-table_1_column_2 is-center ">
                <div class="cell">男</div>
              </td>
              <td rowspan="1" colspan="1" class="el-table_1_column_3 is-center ">
                <div class="cell">1980/10/01</div>
              </td>
              <td rowspan="1" colspan="1" class="el-table_1_column_4 is-center ">
                <div class="cell">12345678901</div>
              </td>
              <td rowspan="1" colspan="1" class="el-table_1_column_5 is-center ">
                <div class="cell">11</div>
              </td>
              <td rowspan="1" colspan="1" class="el-table_1_column_6 is-center ">
                <div class="cell">edstars@100tal.com</div>
              </td>
              <td rowspan="1" colspan="1" class="el-table_1_column_7 is-center ">
                <div class="cell">北京</div>
              </td>
              <td rowspan="1" colspan="1" class="el-table_1_column_8 is-center ">
                <div class="cell"><button type="button" class="el-button el-button--primary el-button--small is-circle">
                    <!----><i class="el-icon-more"></i>
                    <!----></button> <button type="button" class="el-button el-button--danger el-button--small is-circle">
                    <!----><i class="el-icon-delete"></i>
                    <!----></button></div>
              </td>
            </tr>
            <tr class="el-table__row el-table__row--striped">
              <td rowspan="1" colspan="1" class="el-table_1_column_1 is-center ">
                <div class="cell">张邦鑫</div>
              </td>
              <td rowspan="1" colspan="1" class="el-table_1_column_2 is-center ">
                <div class="cell">男</div>
              </td>
              <td rowspan="1" colspan="1" class="el-table_1_column_3 is-center ">
                <div class="cell">1980/10/01</div>
              </td>
              <td rowspan="1" colspan="1" class="el-table_1_column_4 is-center ">
                <div class="cell">12345678901</div>
              </td>
              <td rowspan="1" colspan="1" class="el-table_1_column_5 is-center ">
                <div class="cell">12</div>
              </td>
              <td rowspan="1" colspan="1" class="el-table_1_column_6 is-center ">
                <div class="cell">edstars@100tal.com</div>
              </td>
              <td rowspan="1" colspan="1" class="el-table_1_column_7 is-center ">
                <div class="cell">北京</div>
              </td>
              <td rowspan="1" colspan="1" class="el-table_1_column_8 is-center ">
                <div class="cell"><button type="button" class="el-button el-button--primary el-button--small is-circle">
                    <!----><i class="el-icon-more"></i>
                    <!----></button> <button type="button" class="el-button el-button--danger el-button--small is-circle">
                    <!----><i class="el-icon-delete"></i>
                    <!----></button></div>
              </td>
            </tr>
            <!---->
          </tbody>
        </table>
        <!---->
        <!---->
      </div>
      <!---->
      <!---->
      <!---->
      <!---->
      <div class="el-table__column-resize-proxy" style="display: none;"></div>
    </div>
  </main>
  <footer class="el-footer" style="height: 60px;">
    <div class="pagination el-pagination"><span class="el-pagination__total">共 12 条</span><span class="el-pagination__sizes"><div class="el-select el-select--mini"><!----><div class="el-input el-input--mini el-input--suffix"><!----><input type="text" readonly="readonly" autocomplete="off" placeholder="请选择" class="el-input__inner"><!----><span class="el-input__suffix"><span class="el-input__suffix-inner"><i class="el-select__caret el-input__icon el-icon-arrow-up"></i><!----><!----><!----><!----><!----></span>
      <!----></span>
      <!---->
      <!---->
    </div>
    <div class="el-select-dropdown el-popper" style="display: none;">
      <div class="el-scrollbar" style="">
        <div class="el-select-dropdown__wrap el-scrollbar__wrap el-scrollbar__wrap--hidden-default">
          <ul class="el-scrollbar__view el-select-dropdown__list">
            <!---->
            <li class="el-select-dropdown__item selected"><span>10条/页</span></li>
            <li class="el-select-dropdown__item"><span>20条/页</span></li>
            <li class="el-select-dropdown__item"><span>30条/页</span></li>
            <li class="el-select-dropdown__item"><span>40条/页</span></li>
          </ul>
        </div>
        <div class="el-scrollbar__bar is-horizontal">
          <div class="el-scrollbar__thumb" style="transform: translateX(0%); webkit-transform: translateX(0%);"></div>
        </div>
        <div class="el-scrollbar__bar is-vertical">
          <div class="el-scrollbar__thumb" style="transform: translateY(0%); webkit-transform: translateY(0%);"></div>
        </div>
      </div>
      <!---->
    </div>
    </div></span><button type="button" disabled="disabled" class="btn-prev"><i class="el-icon el-icon-arrow-left"></i></button>
    <ul class="el-pager">
      <li class="number active">1</li>
      <!---->
      <!---->
      <li class="number">2</li>
    </ul><button type="button" class="btn-next"><i class="el-icon el-icon-arrow-right"></i></button><span class="el-pagination__jump">前往<div class="el-input el-pagination__editor is-in-pagination"><!----><input type="number" autocomplete="off" min="1" max="2" class="el-input__inner"><!----><!----><!----><!----></div>页</span></div>
  </footer>
</section>
`;
